release/__init__.py

```diff
@@ -735,8 +735,16 @@ def create(self, repository_path, channel, tag):
 
         # Metadata should already have things like bootstrap_id in it.
         assert 'bootstrap_dict' in metadata
+        assert 'complete_dict' in metadata
         assert 'commit' in metadata
 
+        # Assert that each variant's bootstrap's active packages are included in its complete package list.
+        for info in metadata['all_completes'].values():
+            bootstrap_active_packages = set(
+                pkgpanda.util.load_json('packages/cache/bootstrap/{}.active.json'.format(info['bootstrap']))
+            )
+            assert bootstrap_active_packages <= set(info['packages'])
```

Comment:
@cmaloney I moved this assertion outside of `make_bootstrap_artifacts()` because now `dcos_installer.backend.do_aws_cf_configure()` is calling it, and that code has no way of knowing the complete package list since it's not yet included in `cluster_packages.json` (blocked on late binding since it blows up CF template size). I don't think this assertion has any value outside of the release code anyway.